ocaml/typing/predef.ml

```diff
@@ -568,6 +564,10 @@ let add_simd_beta_extension_types add_type env =
                 Jkind.Const.Builtin.vec128.jkind)
       ~jkind_annotation:Jkind.Const.Builtin.vec128
 
+let add_simd_beta_extension_types add_type env =
+  let _add_type = mk_add_type add_type in
+  env
+
```

Comment:
Let's just delete this and the corresponding call from `env.ml`.  We can add it back if needed in the future.

Reply:
Sure, deleted